### PR TITLE
chore: move example packages into their own directory

### PR DESCRIPTION
**Pull request by petebacondarwin**
These examples will never be published, and it makes more sense for them to be stored separately from the other packages.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Until we have deccent coverage, and are roughly feature complete, 
-# I (Sunil) would like to approve every PR before it lands. 
+# Until we have decent coverage, and are roughly feature complete,
 
-*       @threepointone
+# I (Sunil) would like to approve every PR before it lands.
```
Comment (threepointone) [inline CODEOWNERS:3]: Add yourself too, and remove this comment

**Review by threepointone — APPROVED** [review]
Thank you for doing this! You Boy Scout you

**Comment by threepointone** [comment]
Why's the delta count so weird? (On phone, so I can't tell)

File: CODEOWNERS (hunk 1 above)
Comment (petebacondarwin) [reply CODEOWNERS]: Done

**Comment by petebacondarwin** [comment]
> Why's the delta count so weird? (On phone, so I can't tell)

Not sure what you mean by delta count... the diffs look right to me.

**Comment by threepointone** [comment]
image

**Comment by petebacondarwin** [comment]
Ah, right. That is the changes to the `package-lock.json`.
In order to reconfigure that file with the new layout, I tried just running `npm i` but that didn't seem to work. So I deleted the `package-lock.json` and re-ran `npm i`. I think that was probably a bad thing to do, although the test are passing 😄 .
Let me try again.

**Comment by threepointone** [comment]
No no, all good if it was intentional. Just checking.

**Comment by petebacondarwin** [comment]
I updated the package-lock.json. The change is now much more manageable. Although, you'll note there are a bunch of `extraneous` properties now.

**Comment by petebacondarwin** [comment]
Screenshot 2022-04-02 at 16 14 45

**Comment by threepointone** [comment]
Looks good! Ship ship ship
